### PR TITLE
Some minor improvements

### PR DESCRIPTION
**Pull request by anehx**
* Better timesheet / attendances styling for small devices
* Better validation for duration and timepicker
* Better highlighting of the selected day in the weekly overview

### CODE REVIEW

File: app/components/sy-timepicker/component.js [hunk 1]
```diff
@@ -70,7 +72,15 @@ export default Component.extend({
    * @private
    */
   _add(h, m) {
-    return moment(this.get('value')).add({ h, m })
+    let base = this.get('value')
+
+    if (!base) {
+      base = [h, m].any(n => n < 0)
+        ? this.get('max').add(1, 'minute')
+        : this.get('min')
```
Comment (czosel) [inline app/components/sy-timepicker/component.js]: I'm not sure if i get this. Why is the base`max + 1min` if `h` or `m` is negative?

Comment (czosel) [reply app/components/sy-timepicker/component.js]: Oh that's to make sure you're not entering "yesterday" by clicking the down arrow, right?

Comment (anehx) [reply app/components/sy-timepicker/component.js]: If we don't have a current value and press arrow down, which either subtracts 15 minutes or one hour, we subtract it from the current max which is normally 23:59 if not given. So we need to add a minute to get a valid value.

Comment (czosel) [reply app/components/sy-timepicker/component.js]: :+1:

**Pull request merged**